src/transformers/models/t5/modeling_t5.py

```diff
@@ -1662,6 +1769,9 @@ def forward(
         output_attentions: Optional[bool] = None,
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
+        encoder_position_ids_dict: Optional[Dict[str, Tuple[torch.LongTensor,str]]] = None,
```

Comment:
What is the shape of this tensor?

Reply:
added a comment:
shape of each LongTensor (num_batches, n_input_tokens)